src/Microsoft.IdentityModel.Tokens/SupportedAlgorithms.cs

```diff
@@ -140,7 +189,10 @@ internal static bool IsSupportedKeyWrapAlgorithm(string algorithm, SecurityKey k
             if (string.IsNullOrEmpty(algorithm))
                 return false;
 
-            if (algorithm.Equals(SecurityAlgorithms.RsaPKCS1, StringComparison.Ordinal)
+            if (key.KeySize < 2048)
+                return false;
+
+            if (   algorithm.Equals(SecurityAlgorithms.RsaPKCS1, StringComparison.Ordinal)
```

Comment:
nit: spacing

Reply:
I wanted to have the 'algorithm' variables line up...